### PR TITLE
feature(streams): upgrade authentication in StreamManager, redis-storage, and redis-streams

### PR DESCRIPTION
**Pull request by pcanto-hopeit**

### CODE REVIEW

File: plugins/build/ci-test-plugins.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ fi
 
 if [ "$1" == "" ] || [ "ops/log-streamer" = "$1" ] ; then
 # ops/log-streamer
-export PYTHONPATH=engine/src/:plugins/storage/fs/src/:plugins/ops/log-streamer/src/ && python3 -m pytest -v --cov-fail-under=90 --cov-report=term --cov=plugins/ops/log-streamer/src/ plugins/ops/log-streamer/test/integration/
+export PYTHONPATH=engine/src/:plugins/storage/fs/src/:plugins/ops/log-streamer/src/ && python3 -m pytest -v --cov-fail-under=85 --cov-report=term --cov=plugins/ops/log-streamer/src/ plugins/ops/log-streamer/test/integration/
```
Comment (leosmerling-hopeit) [inline plugins/build/ci-test-plugins.sh:51]: Revert to 90, this passes on CI.

File: plugins/ops/log-streamer/test/integration/test_it_log_file_handler.py [hunk 2]
```diff
@@ -10,10 +10,10 @@
 
 
 @pytest.mark.asyncio
-@pytest.mark.skipif(
-    platform.system().lower() != "linux",
-    reason="LogFileReader uses watchdog that works with no additional helpers only in Linux"
-)
+# @pytest.mark.skipif(
```
Comment (leosmerling-hopeit) [inline plugins/ops/log-streamer/test/integration/test_it_log_file_handler.py:13]: Why this is commented out?

File: plugins/ops/log-streamer/test/integration/test_it_log_reader.py [hunk 3]
```diff
@@ -29,10 +28,6 @@ async def write_logs(log_config, raw_log_entries):
 
 
 @pytest.mark.asyncio
-@pytest.mark.skipif(
```
Comment (leosmerling-hopeit) [inline plugins/ops/log-streamer/test/integration/test_it_log_reader.py:32]: Why this is removed now?

**Review by leosmerling-hopeit — APPROVED** [review]
@pcanto-hopeit please check my comments, increase version and add release notes before merging.

File: plugins/build/ci-test-plugins.sh (hunk 1 above)
Comment (pcanto-hopeit) [reply plugins/build/ci-test-plugins.sh:51]: Done!

File: plugins/ops/log-streamer/test/integration/test_it_log_file_handler.py (hunk 2 above)
Comment (pcanto-hopeit) [reply plugins/ops/log-streamer/test/integration/test_it_log_file_handler.py:13]: Commented out because tests now run on macOS

**Pull request merged**